temba/campaigns/migrations/0043_alter_campaignevent_message.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.16 on 2023-01-27 18:00
+
+from django.db import migrations
+import temba.utils.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("campaigns", "0042_alter_campaignevent_message"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="campaignevent",
+            name="message",
+            field=temba.utils.models.TranslatableField(max_length=640, null=True),
```

Comment:
Those message/text fields should be set to max_length 2000, I think we have added it in a previous fix.